app/src/main/java/ch/epfl/qedit/model/StringPool.java

```diff
@@ -12,17 +12,17 @@
 public class StringPool implements Serializable {
 
     public static final String TITLE_ID = "title";
-    public static final String NO_QUESTION_TITLE_ID = "noQuestionTitle";
-    public static final String NO_QUESTION_TEXT_ID = "noQuestionText";
 
     private Map<String, String> stringPool;
+    private String languageCode;
 
     public StringPool() {
         stringPool = new HashMap<>();
     }
 
     public StringPool(Map<String, String> stringPool) {
         this.stringPool = new HashMap<>(stringPool);
+        this.languageCode = languageCode;
```

Comment:
I don't understand. I think it is equivalent to:

`this.languageCode = this.languageCode;`

Reply:
Oh, that's an artefact that I should have removed. I'm going to deleted the line right away.